mappings/net/minecraft/client/gui/ingame/GuiStructureBlock.mapping

```diff
@@ -1,11 +1,12 @@
-CLASS none/bhx net/minecraft/client/gui/screen/ingame/GuiStructureBlock
+CLASS none/bhx net/minecraft/client/gui/ingame/GuiStructureBlock
 	FIELD f LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD g te Lnone/arv;
 	METHOD <init> (Lnone/arv;)V
 		ARG 0 te
 	METHOD a handleKeyPress (CI)V
-	METHOD a drawScreen (IIF)V
+	METHOD a draw (IIF)V
 	METHOD a handleMouseClick (III)V
 	METHOD a handleButtonPressed (Lnone/bei;)V
```

Comment:
General comment - if we have handleKeyPress, shouldn't this be handleButtonPress?